example/server-multi-dir-list.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const path = require('path')
+const Handlebars = require('handlebars')
+
+const fastify = require('fastify')({ logger: { level: 'trace' } })
+
+const template = `
+<html><body>
+
+dirs
+
+<ul>
+{{#dirs}}
+<li><a href="{{href}}">{{name}}</a></li>
+{{/dirs}}
+</ul>
+
+list
+
+<ul>
+{{#files}}
+<li><a href="{{href}}" target="_blank">{{name}}</a></li>
+{{/files}}
+</ul>
+</body></html>
+`
```

Comment:
formatting: can you please add some indentation to this so it looks more like a properly formatted html file? also, do we need to have it inline or would it make sense to have it as a file on the file system?

Reply:
I think we should keep it as a single file as it makes for an easy all in one reference. I have formatted the handlebar template in this example and `server-dir-list.js` to make them easier to read.